zookeeper-api/src/main/java/org/apache/helix/zookeeper/api/client/RealmAwareZkClient.java

```diff
@@ -71,10 +71,14 @@
   // listener subscription
   List<String> subscribeChildChanges(String path, IZkChildListener listener);
 
+  List<String> subscribeChildChanges(String path, IZkChildListener listener, boolean skipWatchingNodeNotExist);
+
   void unsubscribeChildChanges(String path, IZkChildListener listener);
 
   void subscribeDataChanges(String path, IZkDataListener listener);
 
+  void subscribeDataChanges(String path, IZkDataListener listener, boolean skipWatchingNodeNotExist);
```

Comment:
Can we add some java doc here to explain the parameters and the differences between two methods with same name?

Reply:
added.